platform/switch_ppc64_linux.h

```diff
@@ -2,6 +2,12 @@
  * this is the internal transfer function.
  *
  * HISTORY
+ * 21-Mar-18  Tulio Magno Quites Machado Filho  <tuliom@linux.vnet.ibm.com>
+ *      Added r30 to the list of saved registers in order to fully comply with
+ *      both ppc64 ELFv1 ABI and the ppc64le ELVv2 ABI, that classify this
```

Comment:
ELFv2

Reply:
Thanks! Fixed.

Reply:
Ooops.  Thanks for fixing and merging it!